rental.rb

```diff
@@ -0,0 +1,26 @@
+require './book'
+require './person'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(book, person, date)
+    @book = book
+    @person = person
+    @date = date
+
+    book.add_rental(self)
+    person.add_rental(self)
+  end
+
+  def book=(book)
+    @book = book
+    book.add_rental(self)
+  end
+
+  def person=(person)
+    @person = person
+    person.add_rental(self)
```

Comment:
- The same thing applies here. I suggest changing this lien with `person.rentals << self`